functionnal_tests/cypress/integration/account/manage_user_space.js

```diff
@@ -0,0 +1,72 @@
+import { SELECTORS as s } from '../../support/generic_selector_commands'
+import baseUser from '../../fixtures/baseUser.json'
+import { PAGES as p } from '../../support/urls_commands'
+
+
+describe('Account page', () => {
+  const createOneSpace = (cy, spaceName) => {
+    cy.visitPage({ pageName: p.ACCOUNT })
+    cy.get('[data-cy=sidebarCreateWorkspaceBtn]')
+      .should('be.visible')
+      .click()
+    cy.contains('.cardPopup__header__title', 'New space')
+    cy.get('.newSpace__input').should('be.visible').type(spaceName)
+    // INFO - MP - 2021-11-17 - Select space type
+    cy.get('.singleChoiceList__item').first().should('be.visible').click()
+    cy.get('.newSpace__button .btn').should('be.visible').click()
+
+    // INFO - MP - 2021-11-17 - Wait for the other part of the popup to load
+    cy.get('.newSpace__button .btn').should('have.length', 2)
+
+    // INFO - MP - 2021-11-17 - Select default role
+    cy.get('.singleChoiceList__item').first().should('be.visible').click()
+    cy.get('.newSpace__button .btn').last().should('be.visible').click()
+  }
+
+  describe('Changing account preferences', () => {
+    describe('Manage user space popup', () => {
+      beforeEach(() => {
+        cy.cancelXHR()
+        cy.resetDB()
+        cy.setupBaseDB()
+        cy.loginAs('administrators')
+        createOneSpace(cy, 'newSpaceName')
+        createOneSpace(cy, 'newSpaceName2')
+        createOneSpace(cy, 'newSpaceName3')
+        cy.visitPage({ pageName: p.ADMIN_USER })
+        cy.get('.adminUser__table__tr__td-link').last()
+          .click()
+        cy.get('[data-cy=menusubcomponent__list__spacesConfig]').should('be.visible').click()
+          .click()
```

Comment:
You have two `.click()` here

```suggestion
        cy.get('[data-cy=menusubcomponent__list__spacesConfig]')
          .should('be.visible')
          .click()
```